client/src/pages/BidderDashboard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const BidderDashboard = () => {
+	return (
+		<main>
+			<h1>Bidder Dashboard</h1>
+			<Link to="/list-tenders">
+				<button>View Tenders</button>
+			</Link>
+		</main>
```

Comment:
It should show the list of tenders not a button to the list of tenders

Reply:
Okay will remove the button